src/test/kotlin/com/faire/detekt/rules/ReturnValueOfLetMustBeUsedTest.kt

```diff
@@ -183,6 +183,22 @@ internal class ReturnValueOfLetMustBeUsedTest {
     assertThat(findings).isEmpty()
   }
 
+  @Test
+  fun `let expression can be used in parameter default values`() {
+    val findings = rule.lint(
+        """
+        fun testFunc(
+            foo: Int?,
```

Comment:
(I couldn't think of a case where it is syntactically-correctly used in a parameter context but does not use the return value that was not already covered by other tests)